draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -566,7 +566,7 @@ The JWT MUST contain the following elements:
 * in the JOSE header,
```

Comment:
```suggestion
* in the JWS header,
```

Reply:
per RFC, it's JOSE header.. https://www.rfc-editor.org/rfc/rfc7515#section-2
per DPoP/convention, JWT header might be ok too..

Reply:
yeah, probably any of them would be ok actually

Reply:
```suggestion
* in the JWT header,
```

Reply:
there are other instances where we use `JOSE header`. so let's decide in this issue comment on the term and i will change everywhere accordignly, if needed.

Reply:
let's just keep `JOSE header`